app/service/rest.py

```diff
@@ -946,6 +948,35 @@ def check_if_reply_to_address_already_in_use(service_id, email_address):
         )
 
 
+@service_blueprint.route('/<uuid:service_id>/returned-letter-statistics', methods=['GET'])
+def returned_letter_statistics(service_id):
+
+    most_recent = fetch_most_recent_returned_letter(service_id)
+
+    if not most_recent:
+        return jsonify({
+            'returned_letter_count': 0,
+            'most_recent_report': None,
+        })
+
+    most_recent_reported_at = datetime.combine(
+        most_recent.reported_at, datetime.min.time()
+    )
+
+    if most_recent_reported_at < midnight_n_days_ago(7):
+        return jsonify({
+            'returned_letter_count': 0,
+            'most_recent_report': most_recent.reported_at.strftime(DATETIME_FORMAT_NO_TIMEZONE),
+        })
+
+    count = fetch_recent_returned_letter_count(service_id)
+
+    return jsonify({
+        'returned_letter_count': count.returned_letter_count,
```

Comment:
Optional: The key could be something more descriptive e.g. `returned_letters_in_last_7_days_count`

Reply:
Keeping it as-is for consistency with the other returned letter endpoint responses.